samples/peripherals/storage/src/main.c

```diff
@@ -42,13 +42,16 @@ static void bm_storage_evt_handler_b(struct bm_storage_evt *evt);
 /* Tracks the number of write operations that are in the process of being executed. */
 static volatile int outstanding_writes;
 
-static struct bm_storage storage_a = {
+static struct bm_storage storage_a;
+static struct bm_storage storage_b;
+
+static struct bm_storage_config storage_config_a = {
 	.evt_handler = bm_storage_evt_handler_a,
 	.start_addr = STORAGE_A_START,
 	.end_addr = STORAGE_A_END,
 };
 
-static struct bm_storage storage_b = {
+static struct bm_storage_config storage_config_b = {
 	.evt_handler = bm_storage_evt_handler_b,
 	.start_addr = STORAGE_B_START,
 	.end_addr = STORAGE_B_END,
```

Comment:
These (the storage config structs) don't have to be global now, right? Could have them in `storage_inits()` just to show that it's okay for them to go out of scope and the storage instance will continue to work fine.

Reply:
Right